docs/AboutUs.md

```diff
@@ -48,12 +48,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Dev Ops + Threading
 
-### James Doe
+### Wei Yutong
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/weiyutong.png" width="200px">
```

Comment:
the image name needs to be `wei-yutong.png` to match your github username, you should be able to rename it easily thru gui or by using the `git mv` command